src/components/cards/CardItem.vue

```diff
@@ -78,17 +91,47 @@ export default {
 		return {
 			menuOpened: false,
 			editing: false,
-			copiedCard: ''
+			copiedCard: '',
+			modalShow: false,
+			selectedBoard: '',
+			selectedStack: ''
 		}
 	},
 	computed: {
 		...mapState({
 			compactMode: state => state.compactMode,
-			showArchived: state => state.showArchived
+			showArchived: state => state.showArchived,
+			currentBoard: state => state.currentBoard
 		}),
 		card() {
 			return this.$store.getters.cardById(this.id)
 		},
+		boards() {
+			return this.$store.getters.boards.filter(board => {
+				return board.id !== this.currentBoard.id
+			})
+		},
+		stacksBySelectedBoard() {
+			if (this.selectedBoard === '') {
+				return []
+			}
+
+			let url = OC.generateUrl('/apps/deck/stacks/' + this.selectedBoard.id)
+			axios.get(url)
```

Comment:
Use try...catch instead of chaining .then and .catch :)
With async/await
This will help you figuring things out :)

Reply:
Also, you should not want to use a request in a computed.
Either load it once on beforeMount and store the results in a local data, or use the dedicated multiselect https://vue-multiselect.js.org/#sub-asynchronous-select